spring-xd-dirt/src/main/java/org/springframework/xd/dirt/plugins/spark/streaming/MessageBusConfiguration.java

```diff
@@ -54,6 +56,10 @@
 	 */
 	static ConfigurableApplicationContext createApplicationContext(final Properties properties) {
 		String transport = properties.getProperty("XD_TRANSPORT");
+		if (properties.getProperty(RABBIT_ACKMODE_PROPERTY) != null) {
+			properties.setProperty(RABBIT_ACKMODE_PROPERTY, "MANUAL");
+		}
```

Comment:
Yeah, we always want to set it. I thought of checking if the property is being used. Since we always set this property (via application.yml), I will remove this check and set it always.